src/main/java/net/sf/jabref/importer/fileformat/BibtexParser.java

```diff
@@ -616,8 +616,7 @@ private String parseTextToken() throws IOException {
                 return token.toString();
             }
 
-            if (Character.isLetterOrDigit((char) character) || (character == ':') || (character == '-') || (character == '_')
-                    || (character == '*') || (character == '+') || (character == '.') || (character == '/') || (character == '\'')) {
+            if (Character.isLetterOrDigit((char) character) || (":-_*+./'".indexOf(character) >= 0)) {
```

Comment:
Maybe replace with Arrays.asList(':', '-', .... ).contains(character). But I'm not sure which one is more readable.